### PR TITLE
Fix the setup-py part of the example.

### PR DESCRIPTION
**Pull request by benjyw**
Required moving the .proto files to be
below the exported target (in the filesystem
sense).

If we support setting a source root on
generated protos in the future, we can
modify this further to demo that.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,3 +14,7 @@ __pycache__/
 
 # Editors
 .idea/
+*.iml
+
+# Local databases.
+*.sqlite3
```
Comment (stuhood) [inline .gitignore:20]: What creates these? Are they created throughout the repository, or only at the root?

Comment (benjyw) [reply .gitignore:20]: *.iml is PyCharm/IntelliJ, and .sqlite3 is from my experiments with adding a Django example.  They are both root-only. It was convenient to .gitignore them now, as I kept accidentally adding them to changes.

Comment (stuhood) [reply .gitignore:20]: If they're root only, then you can add a `/` on the front to prevent them from being applied elsewhere.

**Comment by Eric-Arellano** [comment]
Thanks! It'd be good to cherry-pick this into the `2.0` branch, please.

File: .gitignore (hunk 1 above)
Comment (Eric-Arellano) [reply .gitignore:20]: This PR uploaded the actual sqlite binary file.

Comment (benjyw) [reply .gitignore:20]: Ooops!

Re the `/` prefix, we still want to ignore these anywhere they might appear, they just happen to be only created in the root in practice.

**Pull request merged**